common/bloodlines/wc_bloodlines.txt

```diff
@@ -350,6 +350,32 @@ lireesa = { # 603 only, no related event for earlier start
 	flags = { historical_bloodline bloodline_hunter_training }
 }
 #Drust, Kvaldir and the Vrykul
+gorak_tul = { # Event-spawned, Drust warband
+	combat_rating = 8
+	monthly_character_prestige = 0.1
+	throsic_opinion = 10
+	tirassian_opinion = -10
+	bloodline_effect_necromancer_training = 1
+
+	inheritance = patrilineal
+	allow_bastards = yes 		# Not like it's going to make any difference, since he's undead
```

Comment:
So, you added bloodline for character that can't have descendants? For what? Meaning of this game design decision eludes me.